Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -162,6 +163,15 @@
 "chat.message.open_file" = "Antippen zum Öffnen";
 "chat.message.open_browser" = "Im Browser öffnen";
 
+// Chat Drawing
+"chat.drawing.errorTitle" = "Oops!"; // TODO
+"chat.drawing.messageTitle" = "You have to draw something before sending it."; // TODO
+"chat.drawing.fileName" = "Drawing"; // TODO
+"chat.drawing.settings.opacity" = "Opacity"; // TODO
+"chat.drawing.settings.width" = "Width"; // TODO
+"chat.drawing.settings.color.selected" = "Selected color"; // TODO
+"chat.drawing.settings.color.others" = "Choose color below:"; // TODO
```

Comment:
Some strings aren't being used anymore, like the opacity, width and color, right?

Reply:
Those are being used here:

<img width="216" alt="zrzut ekranu 2018-02-27 o 18 15 33" src="https://user-images.githubusercontent.com/3925166/36743305-40abe0fc-1bea-11e8-8647-b445ff8c3754.png">
<img width="213" alt="zrzut ekranu 2018-02-27 o 18 15 39" src="https://user-images.githubusercontent.com/3925166/36743306-40cd3b12-1bea-11e8-9856-de0dab5a196b.png">

Should I remove those? Also, I will change opacity to be percentage (will be better I think)

Reply:
No!!! Keep it, please. I forgot that they were being used here.

Reply:
Yes, percentage sounds better to me also.

Reply:
"chat.upload.draw" = "Bild zeichnen";
"chat.drawing.errorTitle" = "Oops!";
"chat.drawing.messageTitle" = "Du musst etwas zeichnen, bevor du es senden kannst.";
"chat.drawing.fileName" = "Zeichnung";
"chat.drawing.settings.opacity" = "Transparenz";
"chat.drawing.settings.width" = "Breite";
"chat.drawing.settings.color.selected" = "Gewählte Farbe";
"chat.drawing.settings.color.others" = "Wähle eine Farbe:";


@rafaelks on web-frontend we changed to the formal "Sie" in german (I'd like to have that switchable), should we change that on mobile too? I liked the informal "Du" more.